hack/jenkins/README.md

```diff
@@ -0,0 +1,27 @@
+# Jenkins
+
+[Jenkins](http://jenkins-ci.org/) is a pluggable continuous
+integration system. The Google team is running a Jenkins server on a
+private GCE instance for the Kubernetes project in order to run longer
+integration tests, continuously, on different providers. Currently, we
+(Google) are only running Jenkins on our own providers (GCE and GKE)
+in different flavors.
+
+## General flow
+The flow of the Google Jenkins server:
+* On a commit, the `kubernetes-build` job runs the `build.sh` script (in this directory), which results in build assets getting pushed to GCS and the `latest.txt` file in the `ci` bucket being updated. That job then triggers `kubernetes-e2e-*`.
```

Comment:
I hope that new commit makes the parallelization and polling about as clear as mud. PTAL.